### PR TITLE
Changes on text UI review

### PR DESCRIPTION
**Pull request by Deren-Liao**
A minor change on List Processes UI.

Some text changes.

### CODE REVIEW

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx [hunk 1]
```diff
@@ -2180,39 +2180,39 @@
     <comment>The message that indicates adding firewall rule is in progress.</comment>
   </data>
   <data name="AttachDebuggerAskToTestConnectivityLaterMessage" xml:space="preserve">
-    <value>Firewall rule may take a few minutes to become effective, click OK to test again.</value>
+    <value>Firewall rule take effect within a few minutes. Click OK to test again.</value>
```
Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: This does not read correctly after removing *may*. It either still needs some modal verb (e.g. *may*, *should* or *will* ), or you should pluralize *rule*: `Firewall rules take effect within a few minutes.`

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx [hunk 2]
```diff
@@ -2180,39 +2180,39 @@
...
+    <value>Google Compute Engine firewall rule blocks port {0} for {1}. Click OK to add a firewall rule that allows traffic to the port.</value>
     <comment>Message displayed if firewall blocks the port.  {0} is the port number. {1} is the port description. </comment>
   </data>
   <data name="AttachDebuggerTestConnectPortMessageFormat" xml:space="preserve">
     <value>Test connecting to {0} at {1}:{2}</value>
     <comment>The message that indicates the test of connecting to remote port is in progress. {0} is the port description, Debugger Remote Tool or Remote PowerShell.  {1} is ip address, {2} is port number.</comment>
   </data>
   <data name="PortManagerRemoteDebuggerDescription" xml:space="preserve">
-    <value>Allow Visual Studio remote debugger port</value>
+    <value>Allow traffic to Visual Studio Remote Debugger port</value>
```
Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Allow traffic *through the* Visual Studio Remote Debugger port

I think the *the* is necessary, and *through* is better than to at describing the relationship of traffic and ports. Does this need a period?

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx [hunk 3]
```diff
@@ -2256,15 +2256,15 @@
...
     <comment>The attach debugger help message first part.  The message is divided to 3 parts --  [first part] [link caption] [last part].</comment>
   </data>
   <data name="AttachDebuggerHelpLastPartMessage" xml:space="preserve">
-    <value>to set up the debugger remote tools properly.</value>
+    <value>setting up the debugger remote tools.</value>
     <comment>The attach debugger help message last part.  The message is divided to 3 parts --  [first part] [link caption] [last part].</comment>
   </data>
   <data name="AttachDebuggerHelpStepLinkCpation" xml:space="preserve">
-    <value>the documentation</value>
+    <value>Learn more about</value>
```
Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Is there more to this string? If not, I might drop the *about*.

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx (hunk 1 above)
Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Change to "rules"

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx (hunk 3 above)
Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Yes, this is part of a long message.

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx (hunk 2 above)
Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: I updated it to follow pattern of other port descriptions.

**Comment by Deren-Liao** [comment]
Updated, please take a look again.

**Comment by Deren-Liao** [comment]
Thanks Jim.

**Pull request merged**